checkstyle/src/glob_patterns.rs

```diff
@@ -25,7 +25,7 @@ fn list_files_in_folder(folder_path: &Path) -> Result<Vec<PathBuf>, std::io::Err
     Ok(file_paths)
 }
 
-fn all_files(folder_path: &Path) -> Vec<PathBuf> {
+pub fn all_files(folder_path: &Path) -> Vec<PathBuf> {
     list_files_in_folder(folder_path).unwrap_or_default()
 }
```

Comment:
Let's put this in a `util.rs` file, since we want to be using it in multiple places. It isn't really related to `glob_patterns` to the extent that it belongs to it.